### ISSUE TITLE
SMDA: AttributeError: 'NoneType' object has no attribute 'items'

### ISSUE THREAD

**Issue by williballenthin** [fireeye/capa#585]
https://github.com/fireeye/capa/pull/568/checks?check_run_id=2686966250

```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tests\fixtures.py:515: in do_test_feature_presence
    features = scope(extractor)
tests\fixtures.py:287: in inner
    f = get_function(extractor, va)
tests\fixtures.py:248: in get_function
    for f in extractor.get_functions():
capa\features\extractors\smda\__init__.py:29: in get_functions
    for function in self.smda_report.getFunctions():
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <smda.common.SmdaReport.SmdaReport object at 0x0000022FBAD11278>

    def getFunctions(self):
>       for _, smda_function in sorted(self.xcfg.items()):
E       AttributeError: 'NoneType' object has no attribute 'items'

c:\hostedtoolcache\windows\python\3.6.8\x64\lib\site-packages\smda\common\SmdaReport.py:115: AttributeError
```

```
FAILED tests/test_smda_features.py::test_smda_features[a1982...-function=0x4014D0-characteristic(cross section flow)-True]
FAILED tests/test_smda_features.py::test_smda_features[al-khaser x64-function=0x14004B4F0-api(__vcrt_GetModuleHandle)-True]
```

**Comment by williballenthin** [comment]
hmm, i'm not able to reproduce this locally

**Comment by williballenthin** [comment]
xfail'd here: https://github.com/fireeye/capa/pull/568/commits/b03d1bdbc9d55327e735bab70786a9b08faa25b6

**Comment by danielplohmann** [comment]
I assume I just fixed this for both test files in smda 1.5.18. 
Usage of struct.pack("L", ...) vs. struct.pack("Q", ...) apparently behaves inconsistently between Windows and Linux. :)

**Comment by mr-tz** [comment]
thanks, works in #615
